mem-27-extra.xml

```diff
@@ -12647,7 +12647,7 @@ Kursivoidun kirjasimen kuvaamiseen voidaan käyttää {chongHa':v@@chong:v:1, -H
     <table name="mem">
       <column name="_id"></column>
       <column name="entry_name">yulyuS qaySar</column>
-      <column name="part_of_speech">n:name,fic</column>
```

Comment:
Do you have _Hamlet_? I don't remember and can't check it now, but I think it says something in there about Julius Caesar which implies he isn't the Roman Emperor from Earth history, but a Romulan or something. But in any case, Anthony and Cleopatra aren't marked fictional, so I guess this change is consistent. OTOH maybe it should be the other way, and Anthony, Cleopatra, and Shakespeare should join Julius Caesar and Hamlet in being "fictional".

I also realised that the Anthony and Cleopatra entries belong in the "extra" file. I'll move them after your commit.

Reply:
Ah, here it is. "Yulyush is the legendary Romulan leader Yulyush Kayshar". So it should be `fic`.

![Screenshot from 2022-07-09 03-56-52](https://user-images.githubusercontent.com/11312380/178087389-7c42dfad-2b9d-42c3-b46e-b16b723b2424.png)

Reply:
Oh, wait, now I remember why **'antonI'** and **tlhI'yopatra'** are in the regular section and not in `extra`. It's because they're from KGT and thus "canon", whereas the names that appear (only) in Hamlet are not. 

**HenrI' vagh** also appears in KGT. 

*Hamlet* and *Shakespeare* appear in KGT only written in English, though I can't see how *Hamlet* could be anything in Klingon other than **Hamlet**.